providers/base/units/suspend/suspend-graphics.pxu

```diff
@@ -24,6 +24,7 @@ template-id: suspend/index_suspend_after_switch_to_card_product_slug_auto
 requires:
  sleep.mem == 'supported'
  rtc.state == 'supported'
+depends: graphics/{index}_auto_switch_card_{product_slug}
 after: graphics/{index}_auto_switch_card_{product_slug}
```

Comment:
Remove this line (as depends -> after)

Reply:
I've removed and the submission is [here](https://certification.canonical.com/hardware/202001-27667/submission/394667/). Thanks.